akka-stream/src/main/scala/akka/stream/scaladsl/StreamConverters.scala

```diff
@@ -182,7 +182,8 @@ object StreamConverters {
             next
           }
         }, 0), false).onClose(new Runnable { def run = queue.cancel() }))
-      .withAttributes(DefaultAttributes.asJavaStream)
+      .named("asJavaStream")
+      .addAttributes(DefaultAttributes.asJavaStream)
```

Comment:
Why some of these got explicit `.named` and others didn't?

Reply:
Some have them in the stage already, but some are actually other stages that get renamed in a factory.